KsApi/ServiceType.swift

```diff
@@ -240,6 +240,8 @@ public protocol ServiceType {
   func fetchProject(projectParam: Param, configCurrency: String?)
     -> SignalProducer<Project.ProjectPamphletData, ErrorEnvelope>
 
+  func fetchProjectPlotData(projectId: Int) -> SignalProducer<ProjectPLOTDataEnvelope, ErrorEnvelope>
```

Comment:
nit: I know we use `PLOT` elsewhere in the app, but I think we should start reducing the acronyms we use to make it easier to understand code context in the future for ourselves and newer engineers that might be onboarded.

```suggestion
  func fetchProjectPledgeOverTimeData(projectId: Int) -> SignalProducer<ProjectPLOTDataEnvelope, ErrorEnvelope>
```